src/warnet/constants.py

```diff
@@ -98,3 +100,10 @@
     f"helm upgrade --install grafana-dashboards {CHARTS_DIR}/grafana-dashboards --namespace warnet-logging",
     f"helm upgrade --install --namespace warnet-logging loki-grafana grafana/grafana --values {MANIFESTS_DIR}/grafana_values.yaml",
 ]
+
+
+INGRESS_HELM_COMMANDS = [
+    "helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx",
```

Comment:
its `ingress-nginx` even for caddy?

Reply:
Yes it is because the ingress controller is provided by nginx and caddy is the reverse proxy. I know I know, layers upon layers but that's how it is!